adapter.yaml

```diff
@@ -1,38 +1,31 @@
-# hostname of a target gNMI server
-host: 127.0.0.1
-# gNMI port
-port: 50061
-# gNMI server username for all gNMI requests
-username: admin
-# gNMI server password for all gNMI requests
-password: admin
-# insecure/certificate-based gNMI server mode for all requests
-insecure: true
+device_config:
+  # hostname of a target gNMI server
+  host: 127.0.0.1
+  # gNMI port
+  port: 50061
+  # gNMI server username for all gNMI requests
```

Comment:
I think generally gNMI works without user and password, we should think about different comment.